### DISCUSSION TITLE
"some values in assay are not integers"

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#121] [Q&A] [answered]
Hello again

trying deseq2 stuff in R and I got an error saying some values in assay are not integers, but the only non-integars in the assay are the headers. How do I bypass this? 

<img width="827" alt="Screen Shot 2021-03-04 at 8 46 17 PM" src="https://user-images.githubusercontent.com/66333757/110068552-d294d600-7d2a-11eb-8453-1eddfb2319da.png">
<img width="889" alt="Screen Shot 2021-03-04 at 8 46 31 PM" src="https://user-images.githubusercontent.com/66333757/110068554-d45e9980-7d2a-11eb-94d0-205bad9e3be9.png">

**Comment by kubu4** [comment]
You'll need to convert all the numbers to integers. Offhand, can't remember R function to do this, but the internet should have a quick answer.

**Comment by aspencoyle** [comment, marked as answer]
If you're still having problems, I used `data <- round(data, digits = 0)`

**Reply by meganewing** [reply]
That worked, thank you again!!
